### PR TITLE
Project documentation

### PR DESCRIPTION
**Pull request by Pica-Eduard**
### Pull Request Overview

This pull request adds the documentation for the TempTune project.

### TODO or Help Wanted

I would apreciate feedback.

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Pica Eduard <eduard_ionut.pica@stud.fils.upb.ro>

### CODE REVIEW

File: docs/project/2024/eduard_ionut.pica/index.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+| [ 18650 LI-ION Battery 3200mA 10A Panasonic]() |  Battery | [23 RON](https://www.emag.ro/acumulator-18650-li-ion-3200ma-10a-panasonic-lincr18650bd/pd/DL86WWBBM/) |
+
+<!-- | []() |  | []() | -->
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/eduard_ionut.pica/index.md:93]: Do you need these libraries?

File: docs/project/2024/eduard_ionut.pica/index.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://example.com)
+2. [link](https://example3.com)
```
Comment (alexandruradovici) [inline docs/project/2024/eduard_ionut.pica/index.md:100]: Please add some links.

File: docs/project/2024/eduard_ionut.pica/index.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+The USB interface connects to the charging module, which is responsible for recharging the battery that powers the entire system. The battery provides power to the Raspberry Pi Pico W and supplies current to the collector terminals of the transistors.
+
+Temperature sensors are used to measure the ambient temperature, and they send their readings to the Pico W microcontroller. When the temperature recorded by a sensor exceeds a predetermined threshold, the Pico W activates the corresponding transistors by sending current to their base terminals.
+
+To operate a 12V cooling fan, a voltage amplifier is employed to boost the output voltage from the Pico W to approximately 12 volts. This elevated voltage drives the fan, enabling it to cool the laptop. The fan continues to operate until the temperature detected by the sensor falls below the set threshold.
+
+The circuit features an on/off button that allows users to power the system up or down, and an LED indicator that signals whether the cooling system is currently active or inactive.
+
+ ![image info](./arhitecture.png)
```
Comment (alexandruradovici) [inline docs/project/2024/eduard_ionut.pica/index.md:39]:
```suggestion
 ![architecture](./arhitecture.png)
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please add your email to the pull request template.

File: docs/project/2024/eduard_ionut.pica/index.md (hunk 1 above)
Comment (Pica-Eduard) [reply docs/project/2024/eduard_ionut.pica/index.md:93]: no we just left them there and were thinking of modifiying it when we got closer to the software milestone

**Comment by alexandruradovici** [comment]
Duplicate of #182.

**Pull request closed without merging**